Common/TableProducer/PID/pidTPCFull.cxx

```diff
@@ -317,26 +334,208 @@ struct tpcPidFullQa {
+                                                     "expected/Ka", "expected/Pr", "expected/De",
+                                                     "expected/Tr", "expected/He", "expected/Al"};
+  static constexpr std::string_view hexpected_diff[Np] = {"expected_diff/El", "expected_diff/Mu", "expected_diff/Pi",
+                                                          "expected_diff/Ka", "expected_diff/Pr", "expected_diff/De",
+                                                          "expected_diff/Tr", "expected_diff/He", "expected_diff/Al"};
+  static constexpr std::string_view hexpsigma[Np] = {"expsigma/El", "expsigma/Mu", "expsigma/Pi",
+                                                     "expsigma/Ka", "expsigma/Pr", "expsigma/De",
+                                                     "expsigma/Tr", "expsigma/He", "expsigma/Al"};
+  static constexpr std::string_view hnsigma[Np] = {"nsigma/El", "nsigma/Mu", "nsigma/Pi",
+                                                   "nsigma/Ka", "nsigma/Pr", "nsigma/De",
+                                                   "nsigma/Tr", "nsigma/He", "nsigma/Al"};
+  static constexpr std::string_view hnsigmapt[Np] = {"nsigmapt/El", "nsigmapt/Mu", "nsigmapt/Pi",
+                                                     "nsigmapt/Ka", "nsigmapt/Pr", "nsigmapt/De",
+                                                     "nsigmapt/Tr", "nsigmapt/He", "nsigmapt/Al"};
+  static constexpr std::string_view hsignal[Np] = {"signal/El", "signal/Mu", "signal/Pi",
+                                                   "signal/Ka", "signal/Pr", "signal/De",
+                                                   "signal/Tr", "signal/He", "signal/Al"};
+
+  HistogramRegistry histos{"Histos", {}, OutputObjHandlingPolicy::QAObject};
+
+  Configurable<int> logAxis{"logAxis", 1, "Flag to use a log momentum axis"};
+  Configurable<int> nBinsP{"nBinsP", 3000, "Number of bins for the momentum"};
+  Configurable<float> minP{"minP", 0.01, "Minimum momentum in range"};
+  Configurable<float> maxP{"maxP", 20, "Maximum momentum in range"};
+  Configurable<int> nBinsDelta{"nBinsDelta", 200, "Number of bins for the Delta"};
+  Configurable<float> minDelta{"minDelta", -1000.f, "Minimum Delta in range"};
+  Configurable<float> maxDelta{"maxDelta", 1000.f, "Maximum Delta in range"};
+  Configurable<int> nBinsExpSigma{"nBinsExpSigma", 200, "Number of bins for the ExpSigma"};
+  Configurable<float> minExpSigma{"minExpSigma", 0.f, "Minimum ExpSigma in range"};
+  Configurable<float> maxExpSigma{"maxExpSigma", 200.f, "Maximum ExpSigma in range"};
+  Configurable<int> nBinsNSigma{"nBinsNSigma", 200, "Number of bins for the NSigma"};
+  Configurable<float> minNSigma{"minNSigma", -10.f, "Minimum NSigma in range"};
+  Configurable<float> maxNSigma{"maxNSigma", 10.f, "Maximum NSigma in range"};
+  Configurable<int> applyEvSel{"applyEvSel", 0, "Flag to apply rapidity cut: 0 -> no event selection, 1 -> Run 2 event selection, 2 -> Run 3 event selection"};
```

Comment:
In line 204 you have set the event selection to 2, so Run 3 and here it is kept at 0, so no event selection. Is this done on purpose for the tpcPidFullQaWTof?

Reply:
I will make it the same

Reply:
Done ;)